package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fast-sourcemap-concat",
-  "version": "2.1.0",
```

Comment:
When we offer the PR upstream it is common to not bump the package number. It is up to the maintainer to do that once a new release is ready.